server/src/main/java/org/apache/iotdb/db/client/ConfigNodeClient.java

```diff
@@ -441,9 +486,13 @@ public TStorageGroupSchemaResp getMatchedStorageGroupSchemas(List<String> storag
           return resp;
         }
       } catch (TException e) {
+        logger.warn(
+            "Failed to connect to ConfigNode {} from DataNode {} when executing {}",
+            configNode,
+            config.getAddressAndPort(),
+            Thread.currentThread().getStackTrace()[1].getMethodName());
         configLeader = null;
       }
-      reconnect();
```

Comment:
Why getMatchedStorageGroupSchemas is different from other interfaces in this line?